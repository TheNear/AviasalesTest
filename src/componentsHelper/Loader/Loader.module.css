.wrapper {
  margin-bottom: 20px;
  padding: 5px 20px;
  display: flex;
  align-items: center;
  justify-content: center;
  background-color: white;
  border-radius: 5px;
  box-shadow: 0px 2px 8px rgba(0, 0, 0, 0.1);
}

.image {
  margin-left: 20px;
  height: 30px;
  width: 30px;
}

.text {
  color: #4A4A4A;
  font-family: inherit;
}