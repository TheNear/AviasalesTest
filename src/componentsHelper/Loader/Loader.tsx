import React from "react";
import style from "./Loader.module.css";
import spinner from "../../assets/images/spinner.svg";

const Loader: React.FC = () => {
  return (
    <div className={style.wrapper}>
      <p className={style.text}>Загружаем билеты...</p>
      <img className={style.image} src={spinner} alt="loader" />
    </div>
  );
};

export default Loader;
