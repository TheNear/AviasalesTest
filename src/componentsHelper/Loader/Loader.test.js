import React from "react";
import { shallow } from "enzyme";
import { Loader } from "./Loader";

describe("Test custom Loader component", () => {
  it("Render empty Loader", () => {
    const wrapper = shallow(<Loader />);
    expect(wrapper).toMatchSnapshot();
  });
});
