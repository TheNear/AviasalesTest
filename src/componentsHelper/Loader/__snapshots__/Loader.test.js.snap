// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test custom Loader component Render empty Loader 1`] = `
<div
  className="wrapper box"
>
  <p
    className="text"
  >
    Загружаем билеты...
  </p>
  <img
    alt="loader"
    className="image"
    src="spinner.svg"
  />
</div>
`;
